terraform/lifecycle.tf

```diff
@@ -0,0 +1,17 @@
+resource "aws_s3_bucket_lifecycle_configuration" "example" {
+  bucket = data.terraform_remote_state.core.outputs.pipelines_bucket
+
+  rule {
+    id = "expire-jobs"
+
+    filter {
+      prefix = "analysis/jobs/"
+    }
+
+    expiration {
+      days = 90
+    }
+
+    status = "Enabled"
+  }
+}
```

Comment:
I guess since this was a static rule, there is not special permissions that need to be set up to allow this, right?  Also, does this work fine for the dev and staging repos as well?  [I don't really understand where the pipelines_bucket value is coming from - some external module?]

Reply:
@danscales yeah, in this case it's just terraform doing it, and terraform has admin access. We needed permission for the Data API because the API is actively changing the lifecycle rules.

Yeah, the pipelines if coming from the core infra (https://github.com/wri/gfw-aws-core-infrastructure), which sets up shared infrastructure like buckets, databases, etc. So in this case I'm adding the rule to a shared bucket, but just to the prefix we use. It worked correctly in dev for me.